From fca4b2a3aab3e55de0c3e4ee473df0c65254b8f1 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Thu, 28 Aug 2014 14:22:29 -0400
Subject: [cpufreq] remove invalid comment from __cpufreq_remove_dev()

Message-id: <1409235879-1283-56-git-send-email-prarit@redhat.com>
Patchwork-id: 89500
O-Subject: [RHEL7.1 PATCH BZ 1134639 055/185] cpufreq: remove invalid comment from __cpufreq_remove_dev()
Bugzilla: 1134369
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Mark Langsdorf <mlangsdo@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1134639

commit 67a29e558b17a923c3a53c348315c572b8ca261a
Author: Viresh Kumar <viresh.kumar@linaro.org>
Date:   Wed Oct 2 14:13:11 2013 +0530

    cpufreq: remove invalid comment from __cpufreq_remove_dev()

    Some section of kerneldoc comment for __cpufreq_remove_dev() is invalid now.
    Remove it.

    Suggested-by: Srivatsa S. Bhat <srivatsa.bhat@linux.vnet.ibm.com>
    Signed-off-by: Viresh Kumar <viresh.kumar@linaro.org>
    Reviewed-by: Srivatsa S. Bhat <srivatsa.bhat@linux.vnet.ibm.com>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Cc: Lenny Szubowicz <lszubowi@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/cpufreq/cpufreq.c b/drivers/cpufreq/cpufreq.c
index 89da332..dfe7968 100644
--- a/drivers/cpufreq/cpufreq.c
+++ b/drivers/cpufreq/cpufreq.c
@@ -1310,8 +1310,6 @@ static int __cpufreq_remove_dev_finish(struct device *dev,
  * __cpufreq_remove_dev - remove a CPU device
  *
  * Removes the cpufreq interface for a CPU device.
- * Caller should already have policy_rwsem in write mode for this CPU.
- * This routine frees the rwsem before returning.
  */
 static inline int __cpufreq_remove_dev(struct device *dev,
            struct subsys_interface *sif,
-- 
1.7.1